app/src/main/java/com/xlythe/math/GraphModule.kt

```
@@ -0,0 +1,96 @@
+            var x = mMinX
+            while (x <= mMaxX) {
+                if (isCancelled()) {
+                    return null
+                }
+
+                try {
+                    mSolver.mSymbols.define("X", x)
+                    val y = mSolver.mSymbols.eval(equation)
+                    series.add(Point(x, y))
+                } catch (e: SyntaxException) {
+                }
```

detekt (1.23.7)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)